frontend/src/components/forms/UserForm.vue

```diff
@@ -35,13 +35,15 @@ import type { IClub } from "@/utils/interfaces/IClub";
 import { districtRole } from "@/utils/types/commonTypes";
 
 /* Types */
-type UserType = "districtAdmin" | "clubUser" | null;
-type formType =
+//newUser = when user is creating account for first time, prospectUser = user pending approval
+export type UserTypeForm = "districtAdmin" | "clubUser" | "newUser" | "prospectUser" | null;
+export type formType =
```

Comment:
yo great job overall modifying this to work for you.

Reply:
Thanks!